sympy/solvers/tests/test_solveset.py

```diff
@@ -3530,3 +3530,14 @@ def test_issue_22628():
 
 def test_issue_25781():
     assert solve(sqrt(x/2) - x) == [0, S.Half]
+
+def stationary_points(function, variable, domain):
```

Comment:
Don't redefine the function to give the old answer, use the SymPy definition of this function and assert that it now gives the correct answer.